Weekly meeting minutes, 2020-03-12 (#747)
HSF/hsf.github.io: organization/_posts/2020/2020-03-12-coordination.md

---
title: "HSF Weekly Meeting #182, 12 March, 2020"
layout: meetings
---

# {{page.title}}

> Present/Contributors: Graeme Stewart, Pere Mato, Eduardo Rodrigues, Ben Morgan, Paul Laycock, Attila Krasznahorkay, Mason Proffitt, Efe Yazgan, Daniel Elvira, Michel Jouvin, Kyle Knoepfel, Teng Jian Khoo, Liz Sexton-Kennedy, Maria Girone, Philippe Canal, Stefan Roiser, Daniel Elvira, Witek Pokorski, Andrea Rizzi, Chris Jones, David Lange, Gloria Corti, Horst Severini, 

Reviewer: ```suggestion
Present/Contributors: Graeme Stewart, Pere Mato, Eduardo Rodrigues, Ben Morgan, Paul Laycock, Attila Krasznahorkay, Mason Proffitt, Efe Yazgan, Daniel Elvira, Michel Jouvin, Kyle Knoepfel, Teng Jian Khoo, Liz Sexton-Kennedy, Maria Girone, Philippe Canal, Stefan Roiser, Witek Pokorski, Andrea Rizzi, Chris Jones, David Lange, Gloria Corti, Horst Severi

ni, 
```

Apologies: Caterina Doglioni, Agnieszka Dziurda

## News, general matters

### HEP-CCE Kick Off Meeting

> There was an [organizing meeting](<https://indico.fnal.gov/event/23262/other-view?view=standard>) for a new project organization of the CCE which is a collaboration of ANL, BNL, Fermilab,and LBNL participants funded to work on HEP's most pressing and uncovered topics.

Reviewer: It would make sense to define CCE since almost nobody knows what it is/means. @sextonkennedy ?

Author: Done

> We've agreed that going forward all technical meetings should fully public in the same way that IRIS-HEP meetings are. It seems to me that this could be coordinated through the HSF sub-group conveners.

Reviewer: ```suggestion
We've agreed that going forward all technical meetings should be fully public in the same way that IRIS-HEP meetings are. It seems that this could be coordinated through the HSF sub-group conveners.
```

> Main topics interesing to HSF:

Reviewer: ```suggestion
Main topics interesting to HSF:
```

-  Portable Parallelization Strategies: meaningful systematic comparison of a few pieces of HEP code (FastCalosim, Patatrack, Wirecell)

>     - Overlap with interests of Framework WG and Reco WG

Reviewer: ```suggestion
    - Overlap with interests of Frameworks WG and Reco & Trigger WG
```

    - Noted that CERN IT are investigating future solutions for video conferencing, not necessarily Vidyo
- Liz:  I second Caterina's suggestion.  Fermilab is using zoom so I have the opportunity to compare both solutions and find zoom superior.
    - Could investigate using FNAL's infrastructure to help us host this event? (Not sure exactly what the requirements would be at the moment, but FNAL has first class connectivity worldwide)
- Horst: Also supportive (and it works better).
- Eduardo: just for completeness, let me say that there is also [SpeakApp](https://speakapp.link), which is used by Belle II. I believe our colleagues are very happy with it. It's all in the browser.

### Alpaka training event

On **Monday 27 April** a training session for Alpaka will be held in the CERN/IT auditorium (modulo COVID-19). Event info and registration at <https://indico.cern.ch/event/883212/>. In case we need to postpone the event all registered participants will be contacted. *Please sign-up, but do not make any travel arrangements.*

### CUDA Training at CERN

> Maria attended a decicated meeting with Nvidia for organising training through hackathons and bootcamps. Want to host at CERN a CUDA dedicated bootcamp, 2-3 days (pre-requisites are to be a decent CPU coder). Then a hackathon as a follow-up, when people have real projects to work on.

Reviewer: ```suggestion
Maria attended a dedicated meeting with Nvidia for organising training through hackathons and bootcamps. Want to host at CERN a CUDA dedicated bootcamp, 2-3 days (pre-requisites are to be a decent CPU coder). Then a hackathon as a follow-up, when people have real projects to work on.
```

- Graeme contacted ATLAS with a view to having a similar discussion in early April

#### HSF Feedback Sessions

- Maybe organise one longer meeting, but also decide in advance what topics need discussion time

Proposal: HSF review and feedback week of 6 April, one afternoon 15-17h (is that enough time?).
- Launch a doodle ASAP, avoid using the Coordination slot

## Working Group Updates

### Data Analysis
* Joint meeting with DOMA on March 17th (<https://indico.cern.ch/event/890991/>), announcement made in the proper lists.
    * First meeting will feature ATLAS+CMS, but more instances envisioned, and have suggestions of collaborations to invite (more ideas welcome!)
* Dumped a fair bit of text into the LHCC draft (4 pages, but can use refinement -- will evolve based on upcoming meetings)

### Detector Simulation
- Next topical meeting planned on 25th March; proposed topic: Needs and experience of non-LHC HEP community; currently contacting potential speakers
- Following meeting planned for 15 or 22 April
- Two pages of LHCC draft document on simulation written, more coming

> - R&D Forum for Simulation on Accelrators? 

Reviewer 2: "Accelerators"
